community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/Operations.java

```diff
@@ -292,6 +295,12 @@ public boolean nodeDelete( long node, boolean lock ) throws AutoIndexingKernelEx
         return false;
     }
 
+    // Assuming that the nodeCursor have been initialized to the node that labels are retrieved from
```

Comment:
Please convert this comment to a javadoc, which is the right way to document assumptions and contracts.

Reply:
OK, this assumption is made everywhere in this class it seems. Perhaps should be a class-level thing?